javascript/javascript1/week1/main.js

```diff
@@ -0,0 +1,46 @@
+/*1. Future age calculator*/
+let yearOfBirth = 1989;
+let yearFuture = 2045;
+let age = yearFuture - yearOfBirth;
+console.log('Your age will be either ' + age + ' or ' + (age - 1));
```

Comment:
You don't need to add (age -1) . Your age is : 2020 - your year of birth for example :). so 
```suggestion
console.log('Your age will be either ' + age );
```